test/api/libs/responseVerifyer.py

```diff
@@ -82,7 +82,7 @@ def verify_endpoint(self, endpoint: str):
 			elif 'application/zip' in content:
 				expected_mimetype = 'application/zip'
 				jsonData = content[expected_mimetype]
-				# Thie endpoint requires HEADER authentication
+				# The endpoint requires HEADER authentication
```

Comment:
Just a comment:

I think the intended text was `This`, but I don't think this will change something here.
The same for line 129.